addons/t9n/static/src/core/projects_list.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<templates xml:space="preserve">
+    <t t-name="t9n.ProjectList">
+        <div class="container">
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>Project Name</th>
+                        <th>Source Language</th>
+                        <th>Target Languages</th>
+                        <th>Resource Count</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <t t-foreach="state.projects" t-as="project" t-key="project.id">
+                        <tr>
+                            <td>
+                                <t t-esc="project.name"/>
+                            </td>
+                            <td>
+                                <t t-esc="project.src_lang"/>
+                            </td>
+                            <td>
+                                <t t-foreach="project.target_langs" t-as="lang" t-key="lang.id">
+                                    <t t-esc="lang.name"/>
+                                    <t t-if="lang !== project.target_langs[project.target_langs.length - 1]">, </t>
```

Comment:
If you want to get the last element of an array, you could use [.at(-1)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at):
```suggestion
                                    <t t-if="lang !== project.target_langs.at(-1)">, </t>
```
But Owl designers are smart and they anticipated that some people might need to check whether an element is the last in the list and created the magic variable [`_last`](https://github.com/odoo/owl/blob/fddb1ec924c5c3a34a10972e71697bbb0abcdd1f/doc/reference/templates.md#loops) for that purpose:
```suggestion
                                    <t t-if="!lang_last">, </t>
```
BUT in your case, you may want to move this logic to a getter on the component so that you can use the formatList helper, which will allow you to translate the `, ` correctly

Reply:
Done!